Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -215,19 +241,17 @@ def Set(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
 def Test(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
     ShowMof('TEST', ConfigType, ConfigID, Contents, Ensure, ContentChecksum)
     retval = 0
-    if not os.path.exists(AGENT_SCRIPT_PATH):
-        LG().Log(LogType.Error, 'npmd set cap script does not exist, exiting test')
+    if not os.path.exists(PATH_SETCAP_SCRIPT):
+        LOG_ACTION.log(LogType.Error, 'npmd set cap script does not exist, exiting test')
```

Comment:
LOG_ACTION pumps to diagnostic via NPM Ruby plugin. The interesting logs which are needed for debug or understanding of the functional state are sent to NPM ruby plugin in omsagent that will send these to respective diagnostic channel (currently heartbeat).

Later on this can post a direct diagnostic channel if present.